src/OrchardCore.Modules/OrchardCore.Lucene/Shapes/LuceneContentPickerShapeProvider.cs

```diff
@@ -16,9 +18,9 @@ public LuceneContentPickerShapeProvider(IStringLocalizer<LuceneContentPickerShap
         }
 
         [Shape]
-        public IHtmlContent ContentPickerField_Option__Lucene(dynamic shape)
+        public IHtmlContent ContentPickerField_Option__Lucene(dynamic Shape)
```

Comment:
The parameter name is case sensitive when binding from the display context.

Reply:
Where?